app/helpers/application_helper.rb

```diff
@@ -44,4 +46,8 @@ def paperclip_full_url(upload)
     "#{Rails.application.config.action_controller.relative_url_root}" \
       "#{upload.url}"
   end
+
+  def env?(var)
```

Comment:
I don't think this belongs in `ApplicationHelper` - Rails helpers, in the formal sense, are designed to provide short methods when rendering views. The fact that we're using this across the entire application, including initializers, seems to indicate strongly that it belongs somewhere else. [This article](http://guides.rubyonrails.org/initialization.html) detailing the Rails initialization process should be helpful, I'm guessing that this should live somewhere in `lib/` - maybe we include it manually in a base initializer that runs before all the others? Let's discuss.